core/pom.xml

```diff
@@ -43,6 +44,11 @@
             <artifactId>mail</artifactId>
             <version>${version.javax.mail.mail}</version>
         </dependency>
+        <dependency>
+            <groupId>joda-time</groupId>
+            <artifactId>joda-time-jsptags</artifactId>
+            <version>1.1.1</version>
```

Comment:
Don't bring this dependency. Its behavior can be implemented using only simple EL.